services/embeddings/health_mappings.py

```diff
@@ -0,0 +1,110 @@
+import os
```

Comment:
This file needs moving into `services/embeddings/health-mappings`

Also, I was actually expecting one folder for `loinc` and another for `snowmed`. One folder per store, rather than lumping them together.

The idea is that loinc and snomed are stores that can be freely searched by other services, and not innately bound to the healthcare mappings project.